code/datums/components/crafting/recipes.dm

```diff
@@ -448,6 +448,18 @@
 	time = 40
 	category = CAT_ROBOT
 
+/datum/crafting_recipe/vim
+	name = "Vim"
+	result = /obj/vehicle/sealed/car/vim
+	reqs = list(/obj/item/clothing/head/helmet/space/eva = 1,
+				/obj/item/bodypart/l_leg/robot = 1,
+				/obj/item/bodypart/r_leg/robot = 1,
+				/obj/item/flashlight = 1,
+				/obj/item/assembly/voice = 1)
+	tool_behaviors = list(TOOL_SCREWDRIVER)
+	time = 6 SECONDS //Has a four second do_after when building manually
+	category = CAT_ROBOT
```

Comment:
@tralezab I do not play is this sane? thx lu

Reply:
if you mean the ingredients, then it takes the same items it would if you wanted to construct it by hand

Reply:
(currently you can build vim in-game by attaching all of the items above to an EVA helmet)